File: Level_1/7_SameOutput/SameOutput.java
import java.util.Scanner;

class SameOutput{
    public static void main(String[] args){
        Scanner scan = new Scanner(System.in);
        String input;
        // int count = 0;

        input = scan.next();
        // if(scan.hasNextLine()){
        //     if(input.length() > 100){
        //         input.substring(100, input.length());
        //     }
        // }
        
        System.out.println(input);
        
        scan.close();
    }
}

File: Level_1/7_SameOutput/SameOutput.cpp
#include <string>
#include <iostream>
using namespace std;

int main(){
    string input;
    int count = 0;

    while(count < 100){
        cin>>input;
        if(input.length() > 100){
            input.erase(100, input.length() - 100);
        }
        cout<<input<<endl;
        count ++;
    }
    return 0;
}